flask_lesson_intro/templates/item.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+    {% for i in data %}
+        {% if i['title'] == item %}
```

Comment:
maybe this logic you should do at the route?

Reply:
please explain what do you mean

Reply:
done